litejpeg/core/huffman/ac_cr_rom.py

```diff
@@ -0,0 +1,39 @@
+from litex.gen import *
+from litex.soc.interconnect.stream import *
+from litejpeg.core.common import *
+
+from litejpeg.core.huffman.tablebuilder import build_huffman_rom_tables
+
+
+def ac_cr_rom(self,address1,address2,data_out_size,data_out_code):
+
+    code, size = build_huffman_rom_tables(
+    '/home/ishan/gsoc/environment/litejpeg-master/litejpeg/core/huffman/ac_cr_rom.csv')
```

Comment:
Umm, this is a path to your computer :-P

You want to use __file__ to find the path of this .py file. See https://docs.python.org/2/library/runpy.html and https://stackoverflow.com/questions/7116889/python-file-attribute-absolute-or-relative

Something like this should work
```python

import os.path

HUFFMAN_DIR = os.path.dirname(__file__)
HUFFMAN_CSV = os.path.join(HUFFMAN_DIR, 'ac_cr_rom.csv')
```

Reply:
Done